alerts/plugins/ipaddr.py

```diff
@@ -0,0 +1,132 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+from configlib import getConfig, OptionParser
+import netaddr
+import os
+
+
+def isIPv4(ip):
+    try:
+        return netaddr.valid_ipv4(ip)
+    except:
+        return False
+
+
+def isIPv6(ip):
+    try:
+        return netaddr.valid_ipv6(ip)
+    except:
+        return False
+
+
+def addError(message, error):
+    '''add an error note to a message'''
+    if 'errors' not in message.keys():
+        message['errors'] = list()
+    if isinstance(message['errors'], list):
+        message['errors'].append(error)
+
+
+class message(object):
+    def __init__(self):
+        '''
+        takes an incoming alert
+        and uses it to trigger an event using
+        the pager duty event api
```

Comment:
Copy/pasta incorrect comments

Reply:
Comments corrected to reflect the reality.